addons/mail/models/mail_thread.py

```diff
@@ -4609,6 +4612,7 @@ def _message_update_content(self, message, body, attachment_ids=None, partner_id
             Store.Many("partner_ids", ["avatar_128", "name"], rename="recipients"),
             "pinned_at",
             "write_date",
+            "link_preview_ids",
```

Comment:
IMO we should not add `link_preview_ids` in the `res` everytime as it's unnecessary on every call. You need it only when they are removed.